src/test/java/uk/nhs/prm/e2etests/performance/RepoInPerformanceTest.java

```diff
@@ -131,19 +125,51 @@ private void assertMessagesAreInTransferCompleteQueue(int numberOfMessagesToBePr
     private void sendMessagesToMq(List<RepoInPerfMessageWrapper> messagesToBeProcessed) {
         var intervalBetweenMessagesSentToMq = getIntervalBetweenMessagesSentToMq();
         try {
-            var inboundQueueFromMhs = new SimpleAmqpQueue(config);
+            var inboundQueueFromMhs = new SimpleAmqpQueue(queuePropertySource, config);
             var messageTemplate = Resources.readTestResourceFileFromEhrDirectory("small-ehr-4MB");
-            var counter = new AtomicReference<>(0);
-            messagesToBeProcessed.forEach(message -> {
+            var counter = new AtomicInteger(0);
+            String smallEhr;
+
+            // TODO PRMT-3574 traditional for loop here that could be refactored
+            for (int i = 0; i < messagesToBeProcessed.size(); i++) {
                 counter.updateAndGet(v -> v + 1);
-                var conversationId = message.getMessage().conversationId();
-                var smallEhr = getSmallMessageWithUniqueConversationIdAndMessageId(messageTemplate, conversationId);
-                message.start();
+                String conversationId = messagesToBeProcessed.get(i).getMessage().conversationId();
+
+                smallEhr = getSmallMessageWithUniqueConversationIdAndMessageId(messageTemplate, conversationId);
+                messagesToBeProcessed.get(i).start();
 
                 System.out.println("Item " + counter.get() + " - sending to mq conversationId " + conversationId);
                 inboundQueueFromMhs.sendMessage(smallEhr, conversationId);
+
                 sleeper.sleep(intervalBetweenMessagesSentToMq);
-            });
+            }
+//            messagesToBeProcessed.forEach(message -> {
```

Comment:
Can we please remove this commented out block of code? We can try go and fix this later but for now we don't want it in main